### PR TITLE
resolves #15

### LINKED ISSUE #15: ZeroDivisionError: division by zero
```
Traceback (most recent call last):
  File "//./twitter_bot/main.py", line 34, in <module>
    m.reply()
  File "/twitter_bot/Tweet/Tweet.py", line 61, in reply
    response = self.__parser()
  File "/twitter_bot/Tweet/Tweet.py", line 57, in __parser
    response = func(query)
  File "/twitter_bot/Commands/stats.py", line 15, in stats
    response = f'➡️ We have seen {faccounts} accounts in-game since we started tracking {days_since_start} days ago. Of which, at least {fbans} accounts have been banned off of the Hiscores. This comes out to at least {round((bans/accounts)*100,2)}% of the Playerbase being banned.'
ZeroDivisionError: division by zero
```

### PR DESCRIPTION
**Pull request by Ferrariic**
Adds return if denominator == 0 for kc and stats.

### CODE REVIEW

**Comment by extreme4all** [comment]
resolves #15

File: twitter_bot/Commands/kc.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def kc(rsn):
             else:
                 manual_incorrect = r['manual']['incorrect_reports']
                 accuracy = " They have a report accuracy of " + str(round((manual_reports-manual_incorrect)/(manual_reports)*100, 2)) + "%"
+                
+            if reports == 0:
+                return
```
Comment (extreme4all) [inline twitter_bot/Commands/kc.py]: we need to return a response pretty sure

File: twitter_bot/Commands/stats.py [hunk 2]
```diff
@@ -6,11 +6,15 @@ def stats(text):
...
+    project_start_timestamp = 1614470400
     accounts = r['total_accounts']
     bans = r['total_bans']
     faccounts = "{:,}".format(accounts)
     fbans = "{:,}".format(bans)
-    days_since_start = int((time.time() - 1614470400)/(60*60*24))
+    days_since_start = int((time.time() - project_start_timestamp)/(60*60*24))
+    
+    if accounts == 0:
+        return
```
Comment (extreme4all) [inline twitter_bot/Commands/stats.py]: we need to return a response pretty sure

**Review by extreme4all — CHANGES_REQUESTED** [review]

File: twitter_bot/Commands/stats.py (hunk 2 above)
Comment (Ferrariic) [reply twitter_bot/Commands/stats.py]: resolved with commit 30797c815611cbc3699e32e2cf032d6198594952

File: twitter_bot/Commands/kc.py (hunk 1 above)
Comment (extreme4all) [reply twitter_bot/Commands/kc.py]: Edit: Response is validated, if Response is None nothing will be send
https://github.com/Bot-detector/Bot-detector-Twitter/blob/d5f10ed98e1b67718eacc9d7faec2dc9673d4438/twitter_bot/Tweet/Tweet.py#L62

File: twitter_bot/Commands/stats.py (hunk 2 above)
Comment (extreme4all) [reply twitter_bot/Commands/stats.py]: Edit: Response is validated, if Response is None nothing will be send
https://github.com/Bot-detector/Bot-detector-Twitter/blob/d5f10ed98e1b67718eacc9d7faec2dc9673d4438/twitter_bot/Tweet/Tweet.py#L62

**Review by extreme4all — APPROVED** [review]
looks good

**Pull request merged**